CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Master (Unreleased)
 
+- Fix false-positive for `RSpec/UnspecifiedException` when a function is literally named `raise_exception`. ([@aarestad])
```

Comment:
Function -> method

is this the only problem this pr fixes?

Reply:
Yeah, I'll remove the reference to the other issue since I don't really know anything about it other than that comment.

Reply:
just confirming once this is landed I'm happy to do a follow-up to get my issue closed by e.g. adding a spec, so I support you pretending it doesn't exist 🙂